src/tfr_mission_control/plugin.xml

```diff
@@ -0,0 +1,19 @@
+<library path="lib/libtfr_mission_control">
+  <class name="tfr_mission_control/MissionControl"
+         type="tfr_mission_control::MissionControl"
+         base_class_type="rqt_gui_cpp::Plugin">
+    <description>
+      A GUI plugin to ...
+    </description>
+    <qtgui>
+      <group>
+        <label>New Group</label>
```

Comment:
We should probably fill these out: even though we probably won't ever see them, just for good practice (although no big deal if it doesn't get changed, so don't let this block the PR).

Reply:
sure I don't pretend to be a gui expert, in fact quite the opposite. I just wanted something that turned on.